db/schema.rb

```diff
@@ -805,6 +849,17 @@
     t.check_constraint "trnless OR trn IS NOT NULL", name: "check_trn_presence"
   end
 
+  create_table "teaching_school_hubs", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "region"
```

Comment:
Do we have a definitive list of regions? I know [Ofsted have a published list](https://www.gov.uk/government/organisations/ofsted/about) - DfE [doesn't appear to have a list](https://github.com/DFE-Digital/dfe-reference-data/tree/main/lib/dfe/reference_data) (at least not in the reference data) but they clearly do have them ([here's a list of **regional** directors](https://www.gov.uk/government/organisations/regional-department-for-education-dfe-directors/about#who-we-are)).

Just wondering if this should be an enumerated type. I don't imagine they'll often change!)

Reply:
See https://github.com/DFE-Digital/register-early-career-teachers-public/pull/1906 for progress on regions